wandb/cli.py

```diff
@@ -445,6 +447,106 @@ def sync(ctx, path, id, project, entity, ignore):
         wandb_run.Run.from_directory(
             path, run_id=id, project=project, entity=entity, ignore_globs=globs)
 
+@cli.group(help="Commands for interacting with artifacts")
+def artifact():
+    pass
+
+@artifact.command(context_settings=CONTEXT, help="Upload an artifact to wandb")
+@click.argument("path")
+@click.option("--name", "-n", help="The name of the artifact to push: project/artifact_name")
+@click.option("--description", "-d", help="A description of this artifact")
+@click.option("--type", "-t", default="dataset", help="The type of the artifact")
+@click.option("--alias", "-a", default=["latest"], multiple=True, help="An alias to apply to this artifact")
+@display_error
+def log(path, name, description, type, alias):
```

Comment:
I could still see using "put" here, if "log" is the action when done by a run. "put" is the action when done by a user....